### ISSUE TITLE
Wildcard exclusion does not work (4.7.1)

### ISSUE THREAD

**Issue by Gittyperson** [meganz/MEGAsync#741]
Hello,

I've added this specific exclusion, to avoid syncing .cfg files within a specific folder (its subfolders, essentially):

`C:\Users\Username\Foldername1\Foldername2\*\*.cfg`

This worked in previous MEGAsync versions but as of 4.7.1 (Windows 10) it does not, and .cfg files within all subfolders (those inside Foldername2) are now being uploaded/synced.

**Comment by rl-mega** [comment]
Hello @Gittyperson, and thanks for the report.

Do you mean that, for instance, `C:\Users\Username\Foldername1\Foldername2\foo.cfg` would get synced? I tried on my side but couldn't reproduce (setting up the syncs and exclusions in v4.7.0 and then updating to v4.7.1 did not lead to any previously ignored files to be synced).
It would help us very much if you can send us your logs (if you also have one with the previous behavior that would be even better) so we can furher investigate.
Would you mind sending them to support@mega.nz ? You can upload them to a MEGA folder and send us the link. If you do, please copy/paste the above message in the email, and mention "Github issue #741" in the subject.

Thanks again and best regards,
Raphaël

**Comment by Gittyperson** [comment]
> Do you mean that, for instance, `C:\Users\Username\Foldername1\Foldername2\foo.cfg` would get synced?

No, you skipped the `\*\` part. Foldername2 contains many folders, and .cfg files in each of these folders are being uploaded, so:
 `C:\Users\Username\Foldername1\Foldername2\AnyNameFolder\AnyName.cfg` is being uploaded.

I will see what I can do with the logs, but please try the above as described.

**Comment by rl-mega** [comment]
Thanks, I just wanted to make sure and not miss any case. I actually tried at various levels and could not reproduce.

**Comment by Gittyperson** [comment]
I think the exclusion still works for downloading i.e. the .cfg files are being uploaded to MEGA on one PC, but MEGAsync will not download them on another PC, with the exact same folder structure, folder names and MEGAsync exclusion.

**Comment by Gittyperson** [comment]
Checking on the Cloud Drive, I noticed two instances of the folder in question. Two folders with the same name next to each other, which was strange. Deleted them and cleared the Bin.

I then noticed on the PC that while this folder was named "Foldername", on the exclusion list it was "FolderName" (capital N). I made sure they were both named exactly the same, and tried again. It now works as expected.

I don't know if the issue was caused by the letter case, the double folders or both. I cannot remember the letter case on the Cloud Drive, since I deleted them quickly and hadn't yet noticed the case issue at that point.
